CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,100 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define dicotic games, games `x` where both players can move from
+every nonempty subposition of `x`. We prove that these games are small, and relate them
+to infinitesimals.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
+  - (Does this hold for small infinitesimal games?)
+- Prove that any short game is an infinitesimal (but not vice versa, consider `ω⁻¹`)
```

Comment:
Wait, this statement is clearly false: `1` is not infinitesimal! Did you mean, every short *dicotic* game? If so, is `ω⁻¹` still a valid counterexample?

Reply:
Thanks! Missed that!